uk.co.compucorp.civicrm.tasksassignments/api/v3/Document.php

```diff
@@ -592,6 +592,11 @@ function civicrm_api3_document_get($params) {
             )
         ));
 
+      // foreach loop will break if doing 'is_count' request
```

Comment:
Why?

Reply:
Because $getResult will be an integer, I've updated the comment